torch/csrc/distributed/rpc/rpc_base.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <torch/csrc/distributed/rpc/message.h>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+// Base class for all RPC request and responses.
+class RpcBase {
```

Comment:
Might need a different name for this (e.g. `RpcCommandBase`). `RpcBase` sounds like a base class for `dist.rpc`.